### PR TITLE
Editoral refinements

### PR DESCRIPTION
**Pull request by hferraio**
editorial - mostly.

### CODE REVIEW

File: _Appendix/revisions.md [hunk 1]
```diff
@@ -61,15 +61,14 @@ _This appendix is informative._ It provides an overview of the changes to FIPS 2
 |||Updated example PIV Card diagrams.|&sect;4|
 |||Linked expiration of content signing certificate with card authentication certificate.|&sect;4|
 |||Revised PIN requirements based on SP 800-63B guidelines.|&sect;4|
-|||Deprecated symmetric card authentication key.|&sect;4|
+|||Deprecated symmetric card authentication key and associated SYM-CAK authentication mechanism.|&sect;4, &sec6|
```
Comment (jricher) [inline _Appendix/revisions.md:64]:
```suggestion
|||Deprecated symmetric card authentication key and associated SYM-CAK authentication mechanism.|&sect;4, &sect;6|
```

File: _FIPS201/requirements.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Each agency's PIV implementation **SHALL** meet the control objectives listed ab
 + An issued credential is not duplicated or forged.
 + An issued credential is not modified by an unauthorized entity.
 
-[^initiation]: The initiation of a background investigation is defined as the submission of an investigative request to the Defense Counterintelligence and Security Agency or other authorized federal investigative service provider.
+[^initiation]: See [CSP]](../references/#ref-CSP) for investigation initiation process.
```
Comment (jricher) [inline _FIPS201/requirements.md:52]: Seems incomplete ... Details about? Information on?

**Review by jricher — CHANGES_REQUESTED** [review]
Fixed a syntax error and had a question on phrasing in one spot, but not critical.

File: _FIPS201/requirements.md (hunk 2 above)
Comment (hferraio) [reply _FIPS201/requirements.md:52]: The commit has the explanation.  Or can add it here too.  Essentially, 'initiation' is  not enough. The service provider has to have scheduled interview as well, as per CSP.

Comment (jricher) [reply _FIPS201/requirements.md:52]: What I was saying is that this is more of a wording thing, like:

```suggestion
[^initiation]: See [CSP]](../references/#ref-CSP) for details about the investigation initiation process.
```

**Pull request merged**